charts/retool/templates/_helpers.tpl

```diff
@@ -305,6 +305,14 @@ Set code executor service name
 {{ template "retool.fullname" . }}-code-executor
 {{- end -}}
 
+{{/*
+Set multiplayer service name
+*/}}
+{{- define "retool.multiplayer.name" -}}
+{{ template "retool.fullname" . }}-multiplayer-ws
+{{- end -}}
```

Comment:
was using this originally to name ports, but port names can be max 15 characters

https://stackoverflow.com/questions/73330773/is-there-any-way-to-disable-or-increase-port-name-length-in-kubernetes

Reply:
IMO it could just be `multiplayer-ws` as port name, since it will already be on a Service object which has the retool.fullname namespace in its own name?

Reply:
i think the service portName just had to have http- prefix, but will just keep as is for consistency

and then you can have "retool.name" be longer than 15 chars i suppose 😅